src/components/Button/index.js

```diff
@@ -1,11 +1,19 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { ButtonS } from './styles';
+import { InlineIcon } from '@iconify/react';
+import { Buttons } from './styles';
 
-const Button = ({ children, buttonType, onClick }) => (
-  <ButtonS buttonType={buttonType} onClick={onClick}>
+const Button = ({ children, buttonType, onClick, icon, iconSmall }) => (
+  <Buttons buttonType={buttonType} onClick={onClick}>
     {children}
-  </ButtonS>
+    {icon ? (
```

Comment:
Why is this logical? on both case arre render the same `<InlineIcon icon={iconSmall} />` or `<InlineIcon icon={iconSmall} />` what is the diference on both properties ?
`{icon ? ( <InlineIcon icon={icon} />) : (''")}` omit the iconSmall validation will have the same resutls no ?

Reply:
you're right, I made that logic when I was dealing with react-icons and when I switched to iconify I forgot to fix it